### ISSUE TITLE
Bitcoin Offline - localbitcoinschain.com is down (502)

### ISSUE THREAD

**Issue by bkawk** [oipwg/oip-hdmw#1]
When running a simple script
```
const HDMW = require('oip-hdmw')
const Wallet = HDMW.Wallet;

const myWallet = new Wallet('carbon panda replace drum guess heart inside useless random bulb hint industry');
const coins = myWallet.getCoins()

console.log(`My Wallets Coins: ${coins}`)
```
as found here https://github.com/bkawk/flo-oip

I get this error
```
error: Unable to discoverChain #0!
Error: Discovery error in _discoverChain #0
Error: Error: discovery failed!
Error: Unable to update Address state in _chainPromise
Error: Error Updating Address State for: 14mKFHzncvUrSKUJJjH9Sb7ub21KyKWfMz
Error: Unable to getAddress: 502 Bad Gateway | <!DOCTYPE html>
<!--[if lt IE 7]> <html class="no-js ie6 oldie" lang="en-US"> <![endif]-->
<!--[if IE 7]>    <html class="no-js ie7 oldie" lang="en-US"> <![endif]-->
<!--[if IE 8]>    <html class="no-js ie8 oldie" lang="en-US"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en-US"> <!--<![endif]-->
<head>
```
Why is the script tying to call http://localbitcoinschain.com/ which is down? The website dependency is listed here https://github.com/oipwg/oip-hdmw/blob/master/src/networks/bitcoin.js#L24

**Comment by OstlerDev** [comment]
It looks like localbitcoinschain.com is down like you said. We currently use them to provide support for Bitcoin Mainnet. Sometimes there can be issues with this provider, and we are currently working on finding a more stable provider. I will leave this issue open, until we are able to fix Bitcoin not resolving.

In the meantime, when you create your Wallet, add the `supported_coins` option and set it to support `flo` and `litecoin`. This will remove `bitcoin` from being loaded, which will stop this error from happening :) You can even add Flo testnet if you would like.

Here is some example code:
```javascript
var myWallet = new Wallet('carbon panda replace drum guess heart inside useless random bulb hint industry', {
    supported_coins: ['flo', 'flo_testnet', 'litecoin']
});
```

**Comment by OstlerDev** [comment]
This should hopefully be solved by this commit here: c36dff9cf385be837e65bcfcdf71b6746c4903b4
